pingcap/dm
# *: request the MariaDB master to send Annotate_rows_log_event back

### What problem does this PR solve? 

- request the MariaDB master to send Annotate_rows_log_event back
- ref #52 

### What is changed and how it works?

- add `BINLOG_SEND_ANNOTATE_ROWS_EVENT` dump flag for `COM_BINLOG_DUMP` command

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)
	 1. enable `binlog-annotate-row-events` on MariaDB master
 	 2. sync data from this master
	 3. observe relay/sync stage keep `running`

Code changes

 - Has persistent data change

Side effects

 - Possible performance regression
